packages/go_router/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 14.3.0
+
+  - Adds support for handling fragments/hashes as a first-party parameter in the GoRouter package.
```

Comment:
I think you can have one entry instead of 3

something like

Adds the fragment support to relevant GoRouter APIs